### PR TITLE
Removing dependency and addition of Wrappers

### PR DESCRIPTION
**Pull request by gklps**
Replacing EnsurityTechnologies dependency and including wrappers to rubixgoplatform

### CODE REVIEW

**Review by Allen-Cherian — APPROVED** [review]

**Review by KiranHRubix — APPROVED** [review]
changes look good , no conflicts and error, lets test and merge

File: client/client.go [hunk 1]
```diff
@@ -5,10 +5,10 @@ import (
 	"net/http"
 	"time"
 
-	srvcfg "github.com/EnsurityTechnologies/config"
 	"github.com/EnsurityTechnologies/helper/jsonutil"
```
Comment (harirubix) [inline client/client.go:8]: missed this

**Review by harirubix — CHANGES_REQUESTED** [review]

**Comment by harirubix** [comment]
fixes #79

File: client/client.go (hunk 1 above)
Comment (gklps) [reply client/client.go]: updated

**Comment by gklps** [comment]
@harirubix pls check again and confirm the merge

**Pull request merged**